tests/datasets/test_etci2021.py

```diff
@@ -67,7 +67,7 @@ def test_getitem(self, dataset: ETCI2021) -> None:
             assert x["mask"].shape[0] == 1
 
     def test_len(self, dataset: ETCI2021) -> None:
-        assert len(dataset) == 2
+        assert len(dataset) == 3
```

Comment:
ETCI2021DataModule uses a 80/20 train/test split. Unfortunately, 80% of 2 is 2, so the test split was empty. Had to increase the size of the dataset by 1 to make the test split non-empty.